src/components/HeaderCell/HeaderCell.tsx

```diff
@@ -53,13 +53,16 @@ export const HeaderCell = <TData, TValue>({
 
     const rowSpan = header.isPlaceholder ? header.getLeafHeaders().length : 1;
 
+    const isSorted = header.column.getIsSorted();
+
     return (
         <th
             className={b('header-cell', getHeaderCellClassModes(header), className)}
             colSpan={header.colSpan > 1 ? header.colSpan : undefined}
             rowSpan={rowSpan > 1 ? rowSpan : undefined}
             onClick={header.column.getToggleSortingHandler()}
             style={getCellStyles(header)}
+            aria-sort={isSorted ? `${isSorted}ending` : undefined}
```

Comment:
Use explicit values instead of concatenation:
```ts
const getAriaSort = (sortDirection: SortDirection | false) => {
    if (!sortDirection) {
        return undefined;
    }

    return sortDirection === 'asc' ? 'ascending' : 'descending';
}

// ...

aria-sort={getAriaSort(header.column.getIsSorted())}
```